src/interface/substrate_query.js

```diff
@@ -487,6 +494,16 @@ module.exports = {
                     return {'error': e.toString()};
                 }
             // System
+            case 'system/account':
+                if (!param2) {
+                    return {'error': 'You did not enter the stash '
```

Comment:
The error message should be `You did not enter the account ID that needs to be queried`

Reply:
Done.